### ISSUE TITLE
FYI conda-forge package available

### ISSUE THREAD

**Issue by hmaarrfk** [wkentaro/labelme#460]
Not sure if you care or like to use conda, but I just finished packaging labelme for conda-forge

https://github.com/conda-forge/labelme-feedstock/

Let me know if you want to be a maintainer there.

**Comment by wkentaro** [comment]
Thanks so much! I like it.

I'm not that familiar about how to release to `conda-forge`. Could you briefly tell me what commands (and which files you edit) do you run to upgrade?
Apparently, below commands should be run as first, maybe?

```
git clone https://github.com/conda-forge/labelme-feedstock.git
cd labelme-feedstock

vim recipe/meta.yaml
```

**Comment by hmaarrfk** [comment]
Conda forge automates most things. It uses conda-build to take a recipe and turn it into a package. 

Finally, it automates this for Mac, Linux, and Windows, and uploads it to an anaconda channel.

Typing 

`conda-build -m .ci-support/os_and_python_version recipe` 

Can be used to test build the package.

You should add the conda forge channel to your conda config first.

**Comment by wkentaro** [comment]
That’s awesome! Thanks for doing this. 
I will update the README once I’ve done testing it.

**Comment by wkentaro** [comment]
Closed via 7c589bce6d58f400335635dd820be873e21b80c1

**Comment by wkentaro** [comment]
Thanks. It worked perfectly.
